front/src/app/models/current.ts

```diff
@@ -0,0 +1,5 @@
+export class Current{
+    temp: number;
```

Comment:
be more explicit : temp = (temporary ? temperature?) It's important because if another dev looks into the code he could be lost.